UI/window-projector.cpp

```diff
@@ -371,7 +373,7 @@ void OBSProjector::UpdateProjectorTitle(QString name)
 
 OBSSource OBSProjector::GetSource()
 {
-	return source;
+	return OBSGetStrongRef(weakSource);
```

Comment:
Wasn't able to find an answer myself. Does this increment the source ref when returning? Seems like it would increment the strong ref then return.

Reply:
No, it does not. No memory leaks occur. It is also done this way in media-controls.cpp. https://github.com/obsproject/obs-studio/blob/2ca0e7cc1af0314d4564f501b956fd58919ac30b/UI/media-controls.cpp#L286